src/Components/Institutions/institutions.scss

```diff
@@ -0,0 +1,112 @@
+.insti-main {
+  // position: relative;
+  min-height: 100vh;
+  height: fit-content;
+  width: 100vw;
+  overflow: auto;
+}
+
+.insti-title-container {
+  width: 100%;
+  padding: 0 10%;
+  margin: 10vh 0 0 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.insti-title {
+  font-size: 3em;
+}
+
+.insti-card-img {
+  height: 4em;
+  width: 4em;
+}
+
+.title-search-box {
+  position: relative;
+  height: 100%;
+  width: 34%;
+}
+
+.title-search-box-search {
+  height: 3.4em;
+  width: 100%;
+  padding: 0.5em;
+  border-radius: 5px;
```

Comment:
Border radius should not use absolute units (mostly nothing should use absolute units) as curvature would then depend on resolution. Fix it, and everywhere similar stuff is used

Reply:
Made necessary changes.